sources/Interop/D2D1/um/d2d1/D2D1_BITMAP_BRUSH_PROPERTIES.Manual.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/d2d1helper.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public partial struct D2D1_BITMAP_BRUSH_PROPERTIES
+    {
+        public static readonly D2D1_BITMAP_BRUSH_PROPERTIES DEFAULT = new D2D1_BITMAP_BRUSH_PROPERTIES(D2D1_EXTEND_MODE.D2D1_EXTEND_MODE_CLAMP, D2D1_EXTEND_MODE.D2D1_EXTEND_MODE_CLAMP, D2D1_BITMAP_INTERPOLATION_MODE.D2D1_BITMAP_INTERPOLATION_MODE_LINEAR);
+
+        public D2D1_BITMAP_BRUSH_PROPERTIES(D2D1_EXTEND_MODE extendModeX, D2D1_EXTEND_MODE extendModeY, D2D1_BITMAP_INTERPOLATION_MODE interpolationMode)
```

Comment:
nit: The header file defines most of these as global helper functions rather than constructors.

I'm fine with them being exposed as constructors but we should still expose them as global helpers as well to make porting easier.